config-service-impl/src/test/java/org/hypertrace/config/service/store/DocumentConfigStoreTest.java

```diff
@@ -72,17 +79,19 @@ class DocumentConfigStoreTest {
   private static Value config1 = getConfig1();
   private static Value config2 = getConfig2();
   private static ConfigResourceContext configResourceContext = getConfigResourceContext();
-  private static Collection collection;
-  private DocumentConfigStore configStore;
-  private Clock mockClock;
-  private FilterBuilder filterBuilder;
 
-  @BeforeEach()
+  private Collection collection;
+  @Mock private Clock mockClock;
```

Comment:
So I added the mockito extension just as a best practice, it's not directly related to the change. But it does lend readability as we need less setup and it will fail if we start mocking things that the test path doesn't touch (you can see some unused mocks I removed in `WriteConfigForUpdateWithUpsertCondition`)